### ISSUE TITLE
Set Plex movie watched when play is added

### ISSUE THREAD

**Issue by leepeuker** [leepeuker/movary#440]
If I watch a movie in Jellyfin, I want the movie to be set as watched in Plex as well.

Workflow:
- movie is watched in Plex
- the play is added to Movary via Plex webhook
- Movary processes the new play and triggers job to sync play with Jellyfin
- the job processing will update the watch status in Jellyfin

Additional:
- a full sync (Movary to Plex) option should be available

Problems:
- is there an API provided by Plex to set the watch state of a movie we can use here?

**Comment by leepeuker** [comment]
This could help: https://github.com/leepeuker/movary/pull/274

**Comment by JVT038** [comment]
Plex has a [scrobble endpoint](https://plexapi.dev/docs/plex/mark-played) which you can use to mark things as watched.

Besides that, what if I manually add a play in Movary; will it automatically scrobble to Jellyfin and Plex to mark them as watched?

**Comment by leepeuker** [comment]
Yes, the idea is that Movary is the central point of truth. When Movary gets a new play and the user has the watch state sync active, Plex/Jellyfin is updated, regardless of how the play is added.

Another question is what about play removals. When a user removes the last play of a movie, should the movie be set as unwatched in Plex/Jellyfin too?

**Comment by JVT038** [comment]
No. Only when there is only one play and that has been removed, then it should be set to unwatched.
If there are multiple plays and only one play is removed, then the movie has still been watched at least once, and therefore it should be set to watched in Jellyfin and Plex as well.

**Comment by leepeuker** [comment]
Yeah you are right, that was what I meant, but thanks for explicitly clarifying 👍
